internal/server/backup_handlers.go

```diff
@@ -13,12 +15,22 @@ import (
 // @Tags     Backup
 // @Produce  json
 // @Param    name query string false "Backup routine name"
+// @Param    from query int false "Lower bound timestamp filter"
```

Comment:
Should've mark it as int64, but I could not find way to do it

Reply:
There is no int64/long type in Swagger. int64 will result in the integer type, too. Do you encounter type incompatibility issues generating the client?

Reply:
I found it!